### PR TITLE
FAQ from "Your First Package"

### PR DESCRIPTION
**Pull request by jukent**

### CODE REVIEW

**Comment by jukent** [comment]
Can I merge this if there are no reviews by the end of tomorrow @NCAR/xdev-education ?

File: site/posts/python-tutorial-seminar-series-spring-2021.ipynb [hunk 1]
```diff
@@ -37,6 +37,13 @@
     "\n",
     "If you would like join the *learnpython@ucar.edu* listserv to be informed of updates and events in the seminar series (including the Zoom meeting information), please let us know by filling out this [Google Form](https://forms.gle/7PpZ5uuUNzsfB27P9)."
    ]
+  },
+  {
+   "source": [
+    "<iframe src=\"https://calendar.google.com/calendar/embed?src=c_krmtmqm6kb5u7ke6t5on9l0rus%40group.calendar.google.com&ctz=America%2FDenver\" style=\"border: 0\" width=\"800\" height=\"600\" frameborder=\"0\" scrolling=\"no\"></iframe>"
```
Comment (kmpaul) [inline site/posts/python-tutorial-seminar-series-spring-2021.ipynb:43]:
```suggestion
    "<iframe src=\"https://calendar.google.com/calendar/embed?src=c_krmtmqm6kb5u7ke6t5on9l0rus%40group.calendar.google.com\" style=\"border: 0\" width=\"800\" height=\"600\" frameborder=\"0\" scrolling=\"no\"></iframe>"
```

**Review by kmpaul — CHANGES_REQUESTED** [review]
Remove timezone info in calendar

**Review by kmpaul — APPROVED** [review]
Thanks, @jukent!

**Pull request merged**